packages/flutter/test/material/banner_theme_test.dart

```diff
@@ -50,21 +50,38 @@ void main() {
 
   testWidgets('Passing no MaterialBannerThemeData returns defaults', (WidgetTester tester) async {
     const String contentText = 'Content';
+    const Key tapTarget = Key('tap-target');
```

Comment:
Same comment for this file regarding pre-existing tests. Keeping the old tests ensures our backwards compatibility promise. Feel free to refactor to re-use code where convenient, but we should add new tests instead of re-writing the old to reflect new behavior.